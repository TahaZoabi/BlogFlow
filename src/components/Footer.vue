<script setup lang="ts">
const time = new Date()
const year = time.getFullYear()
</script>

<template>
  <div class="flex flex-col items-center justify-center gap-4">
    <ul class="flex justify-center items-center gap-5">
      <li>About</li>
      <li>Privacy Policy</li>
      <li>Terms of Service</li>
    </ul>
    <h5>Â© {{ year }} Taha Zoabi. All rights reserved.</h5>
  </div>
</template>
