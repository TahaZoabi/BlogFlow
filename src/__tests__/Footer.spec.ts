import { describe, it, expect } from 'vitest'
import { mount } from '@vue/test-utils'
import Footer from '../components/Footer.vue'

describe('Footer Component', () => {
  const wrapper = mount(Footer)
  it('should render correctly', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
