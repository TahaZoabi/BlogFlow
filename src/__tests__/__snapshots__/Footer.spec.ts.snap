// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer Component > should render correctly 1`] = `
"<footer class="flex flex-col items-center justify-center gap-4">
  <h5 class="text-md sm:text-lg">Â© 2024 Taha Zoabi. All Rights Reserved.</h5>
</footer>"
`;
